### DISCUSSION TITLE
Bookmark icon customization

### DISCUSSION THREAD

**Discussion by Joey-Wiley** [zen-browser/desktop#2859] [Ideas]
If you have multiple bookmarks from the same website, they often have the same favicon as their icon. It would be nice if you were able to customize this icon with emojis and/or custom icons.

The default icon could still be the favicon of the website, but being able to customize it after creation would be sweet.

**Comment by N1N74** [comment]
Would be amazing if we could give folders icons also. 
Another use-case for custom bookmark items is when using bookmarklets/sites without icons, that are saved to toolbar, having custom icons without names saves space.

E: this is a popular request for firefox, has gained 90+ upvotes on their site: https://connect.mozilla.org/t5/ideas/be-able-to-change-custom-icons-for-bookmarks/idi-p/5886/
